docs/tutorials/comparison.md

```diff
@@ -79,302 +123,753 @@ source1 = "FirstSourceName"
 
-def zero_pad_and_clean_file_names():
-    current_directory = os.getcwd()
-    for filename in os.listdir(current_directory):
-        base_name, extension = os.path.splitext(filename)
-        parts = base_name.split('_')
+### Basic Keybinds
 
-        if len(parts) > 2 and parts[0].isdigit():
-            parts[0] = parts[0].zfill(5)
-            parts[2] = parts[2].replace("b'", "").replace("'", "")
-            new_filename = '_'.join(parts) + extension
-            os.rename(os.path.join(current_directory, filename), os.path.join(current_directory, new_filename))
+Key              | Action
+-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`Left arrow`     | Move back *n* frames based on [config](#change-frame-increment) (default: *n = 1*)
+`Right arrow`    | Move forward *n* frames based on [config](#change-frame-increment) (default: *n = 1*)
+Number keys      | Switches to source *n* (e.g. `2` switches to `clip2`)
+`Shift` + `S`    | Take and save screenshot of the current frame
+`Ctrl` + `Space` | Mark current frame number for [automatic](#automatic-badge-variant-secondary-text-fastest-option)/[semi-automatic](#semi-automatic-badge-icon-variant-primary-text-recommended) comparisons
 
-# Usage example
-zero_pad_and_clean_file_names()
-```
+### Process
+
+VSPreview offers three methods for creating comparisons:
+
+==- Automatic [!badge variant="secondary" text="Fastest Option"]
+
+Automatic comparisons are created completely without any additional user input. VSPreview will automatically select, capture, and upload frames for you. *This is the fastest method for creating comparisons.*
+
+#### Capturing
+
+1. In VSPreview, navigate to the bottom bar and toggle the *Comp* section
+
+2. Set your parameters. The following parameters should be filled:
+   ![VSPreview comp bar](/static/comparison/vsp-comp.png)
+
+   Key | Description
+   ----|-----------------------------------------------------------------------------------------------------
+   1   | The title of your comparison/show
+   2   | The random frame interval to capture. *This should be set to a value higher or equal to 40 frames*
+   3   | The picture type
+   4   | The [TMDB ID](https://www.themoviedb.org) for the show
+
+   - *See [Comp](#comp) for more detail on what each one is used for*
+
+3. Hit the *Start Upload* button under *Comp* to begin creating your comparison
+
+==- Semi-automatic [!badge icon=":heart:" variant="primary" text="Recommended"]
+
+Semi-automatic comparisons are created with minor user input. VSPreview will automatically capture and upload frame manually marked by the user. *This is the recommended method for creating comparisons.*
+
+#### Setup
+
+1. Locate the frame(s) you want to compare
+   - By default, `Left arrow` and `Right arrow` navigates between frames
```

Comment:
you can go to playback -> and specify the frame skip count. Say 60 frames. then SHIFT + RIGHT/LEFT Arrow to skip 60.

![](https://i.imgur.com/M4FwPKh.png)